### PR TITLE
Added new format for future dates.

### PR DESCRIPTION
**Pull request by Blizzardo1**
Made some code fixes and added a new format checker. See #7 for details.

### CODE REVIEW

File: Roni.Corona/Roni.Corona.DataIngestion/ParseExtensions.cs [hunk 1]
```diff
@@ -2,11 +2,11 @@
 {
     public static class ParseExtensions
     {
+        // No need of using bool to return if parsing succeeded as if fail, returns 0 anyways.
```
Comment (caveofjulian) [inline Roni.Corona/Roni.Corona.DataIngestion/ParseExtensions.cs]: remove comment, this is not the place to discuss it

File: Roni.Corona/Roni.Corona.DataIngestion/Integrations/CsseIntegration.cs [hunk 2]
```diff
@@ -2,42 +2,36 @@
...
-        public CsseIntegration(HttpClient client)
-        {
+        public CsseIntegration(HttpClient client) {
             _client = client;
         }
 
-        public async Task<Dictionary<DateTime, string>> GetNewContent(DateTime lastUpdated)
-        {
-            Dictionary<DateTime, string> csvData = new Dictionary<DateTime, string>();
+        // FIXED: Optimized the method to used Flurl because it's better
```
Comment (caveofjulian) [inline Roni.Corona/Roni.Corona.DataIngestion/Integrations/CsseIntegration.cs]: remove comment, comments are not used to discuss stuff

File: Roni.Corona/Roni.Corona.DataIngestion/Ingester.cs [hunk 3]
```diff
@@ -71,23 +65,32 @@ private Cases MapStringToCases(int[] indices, string content, DateTime date)
...
-            var cells = content.Split(",");
+            string[] cells = content.Split(",");
+            
+            bool isDateParsed = DateTime.TryParseExact(cells[indices[2]], "d/MM/yyyy HH:mm", null,DateTimeStyles.AllowWhiteSpaces, out DateTime lastDate);
+            if (!isDateParsed) lastDate = date;
 
-            var confirmed = cells[indices[3]].ParseToInt();
-            var deaths = cells[indices[4]].ParseToInt();
-            var recovered = cells[indices[5]].ParseToInt();
+            // Proper use of TryParse because if it's not a number, then use last return statement
```
Comment (caveofjulian) [inline Roni.Corona/Roni.Corona.DataIngestion/Ingester.cs]: remove comment, comments are not used to discuss proper use

File: Roni.Corona/Roni.Corona.DataIngestion/Integrations/CsseIntegration.cs [hunk 4]
```diff
@@ -2,42 +2,36 @@
 using System.Collections.Generic;
 using System.Net.Http;
 using System.Threading.Tasks;
+using Flurl;
```
Comment (caveofjulian) [inline Roni.Corona/Roni.Corona.DataIngestion/Integrations/CsseIntegration.cs]: Let's actually not go down this path. Adding dependencies is a drama, and there is no reason at this point to use an external library for a simple integration nor will we have more integrations in this project

**Review by caveofjulian — CHANGES_REQUESTED** [review]
Remove 3d party dependency and comments that were not supposed to be written

File: Roni.Corona/Roni.Corona.DataIngestion/Integrations/CsseIntegration.cs (hunk 4 above)
Comment (Blizzardo1) [reply Roni.Corona/Roni.Corona.DataIngestion/Integrations/CsseIntegration.cs]: This has been completed.

File: Roni.Corona/Roni.Corona.DataIngestion/Ingester.cs (hunk 3 above)
Comment (Blizzardo1) [reply Roni.Corona/Roni.Corona.DataIngestion/Ingester.cs]: This has been completed.

File: Roni.Corona/Roni.Corona.DataIngestion/Integrations/CsseIntegration.cs (hunk 2 above)
Comment (Blizzardo1) [reply Roni.Corona/Roni.Corona.DataIngestion/Integrations/CsseIntegration.cs]: This has been completed.

File: Roni.Corona/Roni.Corona.DataIngestion/ParseExtensions.cs (hunk 1 above)
Comment (Blizzardo1) [reply Roni.Corona/Roni.Corona.DataIngestion/ParseExtensions.cs]: This has been completed.

**Review by caveofjulian — APPROVED** [review]

**Pull request merged**